cats-backend/src/modules/cats/cats/cats.service.ts

```diff
@@ -28,25 +29,26 @@ export class CatsService {
       const matchingMice = await this.mouseModel.findAll({
         attributes: ['catId'],
         where: {
-          name: { [Op.iLike]: mouseName },
+          name: { [Op.iLike]: `%${mouseName}%` },
```

Comment:
```suggestion
          name: { [Op.iLike]: `%:mouseName%` },
```

Then pass the `mouseName` with sequelize replacement